docs/hardware/common.md

```diff
@@ -158,11 +166,42 @@ This metric is [recommended][MetricRecommended].
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
 | [`hw.id`](/docs/attributes-registry/hardware.md) | string | An identifier for the hardware component, unique within the monitored host | `win32battery_battery_testsysa33_1` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| [`hw.type`](/docs/attributes-registry/hardware.md) | string | Type of the component | `battery`; `cpu`; `disk_controller` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| [`hw.error.type`](/docs/attributes-registry/hardware.md) | string | The type of error encountered by the component | `` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`hw.type`](/docs/attributes-registry/hardware.md) | string | Type of the component [1] | `battery`; `cpu`; `disk_controller` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`error.type`](/docs/attributes-registry/error.md) | string | Describes a class of error the operation ended with. [2] | `timeout`; `java.net.UnknownHostException`; `server_certificate_invalid`; `500` | `Recommended` | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 | [`hw.name`](/docs/attributes-registry/hardware.md) | string | An easily-recognizable name for the hardware component | `eth0` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`hw.parent`](/docs/attributes-registry/hardware.md) | string | Unique identifier of the parent component (typically the `id` attribute of the enclosure, or disk controller) | `dellStorage_perc_0` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
+**[1]:** Describes the category of the hardware component for which `hw.state` is being reported. For example, `hw.type=temperature` along with `hw.state=degraded` would indicate that the temperature of the hardware component has been reported as `degraded`.
+
+**[2]:** The `error.type` SHOULD be predictable, and SHOULD have low cardinality.
+
+When `error.type` is set to a type (e.g., an exception type), its
+canonical class name identifying the type within the artifact SHOULD be used.
+
+Instrumentations SHOULD document the list of errors they report.
+
+The cardinality of `error.type` within one instrumentation library SHOULD be low.
+Telemetry consumers that aggregate data from multiple instrumentation libraries and applications
+should be prepared for `error.type` to have high cardinality at query time when no
+additional filters are applied.
+
+If the operation has completed successfully, instrumentations SHOULD NOT set `error.type`.
+
+If a specific domain defines its own set of error identifiers (such as HTTP or gRPC status codes),
+it's RECOMMENDED to:
+
+* Use a domain-specific attribute
```

Comment:
Given the description of the `error.type` attribute, I suggest that we enforce `error.type="hardware"` (or `error.type="hw"`?) and let the instrumentation use `hw.error.type` to specify the type of hardware error.

In issue #940, @eero-t suggested to add another attribute `hw.error.category`, to define even more precisely the type of error.

Reply:
later error.type would have precise specific values i.e. https://opentelemetry.io/docs/specs/semconv/system/hardware-metrics/#hwcpu---physical-processor-metrics or https://opentelemetry.io/docs/specs/semconv/system/hardware-metrics/#hwgpu---gpu-metrics so `error.type` will be defined for specific hardware component

Reply:
> In issue #940, @eero-t suggested to add another attribute `hw.error.category`, to define even more precisely the type of error.

I changed my suggestion later to: https://github.com/open-telemetry/semantic-conventions/issues/940#issuecomment-2085109515
* `.severity`: correctable, uncorrectable (data loss), fatal (functionality loss)
* `.type`: type / category of the error; timeout, memory-parity-error, hw-programming-error etc
* `.location` / `.origin` / `.cause` / `.whatever`: e.g. http-endpoint, display (part of GPU with some memory), compute (GPU pipeline setting)

Reply:
This type of changes should be an additional PR after this one gets merged. So we don't mix refactoring (with minor changes) and major changes/new functionality